sdks/python/apache_beam/typehints/batch_test.py

```diff
@@ -54,6 +64,17 @@
         'element_typehint': str,
         'batch': ["foo" * (i % 5) + str(i) for i in range(1000)],
     },
+    {
+        'batch_typehint': torch.Tensor,
+        'element_typehint': torch.Tensor,
```

Comment:
A single element of a `Tensor` array is also a `Tensor`. e.g.
```
>>> a = torch.Tensor([1,2,3])
>>> a
tensor([1., 2., 3.])
>>> a[0]
tensor(1.)
>>> type(a)
<class 'torch.Tensor'>
>>> type(a[0])
<class 'torch.Tensor'>
```

Reply:
Interesting. It could be problematic to allow this as an element type though, since it's unclear what data type is.

For now, could we always represent scalars as a 0-dim PytorchTensor? i.e. `PytorchTensor[torch.int32, (,)]`. There could also be a wrapper for this, like `PytorchScalar[torch.int32]`

Reply:
Yeah, let me try that